server/index.js

```diff
@@ -18,3 +20,20 @@ app.get('/reviews/:id', (req, res) => {
       console.log(err);
     });
 });
+
+const jsonParser = bodyParser.json();
+app.patch('/review', jsonParser, (req, res) => {
+  console.log(req.body.reviewId);
+  console.log(req.body.likes_count);
+  review.findOneAndUpdate(
+    { _id: req.body.reviewId },
+    { likes_count: Number(req.body.likes_count) + 1 },
+    { useFindAndModify: false },
+  ).exec()
+    .then(() => {
+      res.end();
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
Another option is to send an error message back to the client if .catch is invoked.

Reply:
That's a good point. Will  include that...